src/stateMatrixParser.cpp

```diff
@@ -0,0 +1,443 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <cmath>
+
+#include "NormalizedCompressionMarkovTable.h"
+#include "stateMatrixParser.h"
+
+
+
+std::vector<Metrics> mutate_state_matrix(std::string state_matrix_string,unsigned int states, unsigned int  alphabet_size,unsigned int numIt, bool print ){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'state_matrix_string' should be passed by const reference.](https://app.codacy.com/app/jorgeMFS/TMCompression/pullRequest?prid=4494377)